torch/csrc/jit/test_jit.cpp

```diff
@@ -960,6 +964,94 @@ void testProto() {
   proto.set_producer_name("foo");
 }
 
+void testCustomOperators() {
```

Comment:
Do things like `std::vector<at::Tensor>` work? I don't see tests. what if someone has a lambda like `[](const std::vector<at::Tensor>& foo)`? Does it handle the const& ref logical correctly, or are we going to ask people to only write vector-by-value functions?